.hound.yml

```diff
@@ -1,5 +1,8 @@
 ruby:
   config_file: .rubocop.yml
+erb:
```

Comment:
We've switched to naming these by linter name (rather than language), so this should either be `erb_lint` or its alias `erblint`.

Reply:
@gylaz Thank! So also the other entries are not valid?

Reply:
For legacy reasons, old keys are still supported. It's just the new keys that are added (and all the new documentation) are now by linter name (given that for certain languages we now support multiple linters).

Reply:
Done, thanks!